src/web/services/student-profile.service.ts

```diff
@@ -15,6 +15,20 @@ export class StudentProfileService {
   constructor(private httpRequestService: HttpRequestService) {
   }
 
+  /**
+   * Gets a student profile by calling API.
```

Comment:
Maybe explain that the parameters are optional and what will happen if they are not provided?